src/UI/Implementation/Component/Glyph/Glyph.php

```diff
@@ -66,6 +82,12 @@ public function __construct($type, $action = null) {
 	public function getType() {
 		return $this->type;
 	}
+	/**
+	 * @inheritdoc
+	 */
+	public function getAriaLabel() {
```

Comment:
because it is only used internally. What would be a use case to expose the label? I do not have a strong opinion in this regard though.